db/theme.js

```diff
@@ -0,0 +1,48 @@
+const Pool = require('pg').Pool;
+const config = require('../config/db_config');
+const testConfig = require('../config/db_test_config');
+
+const pool = new Pool({
+    user: config.user,
+    host: config.host,
+    database: config.database,
+    password: config.password,
+    port: config.port,
+});
+
+let getAllThemes = () => {
+    return pool.query('select * from theme');
+};
+
+let getTheme = (id) => {
+    return pool.query('select * from theme where $1 = id', [id]);
+};
+
+let postTheme = (theme) => {
+    let text = "INSERT INTO theme(themename, begindate, enddate, colors) VALUES($1, $2, $3, $4)";
```

Comment:
Chose between capitalized SQL or not (I'd recommand to capitalize every sentences)